### PR TITLE
fix: add ToString overrides where appropriate

### PR DESCRIPTION
**Pull request by pgautier404**

### CODE REVIEW

File: src/Momento.Sdk/Responses/CacheDeleteResponse.cs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 using Momento.Sdk.Exceptions;
 
-public class CacheDeleteResponse
+public abstract class CacheDeleteResponse
```
Comment (malandis) [inline src/Momento.Sdk/Responses/CacheDeleteResponse.cs:5]: Why the abstract modifier?

**Review by malandis — COMMENTED** [review]
Looks good. Just one question

File: src/Momento.Sdk/Responses/CacheDeleteResponse.cs (hunk 1 above)
Comment (pgautier404) [reply src/Momento.Sdk/Responses/CacheDeleteResponse.cs:5]: All of the parent response classes are supposed to be abstract so they can't be instantiated. I just missed this one on the first go.

Comment (malandis) [reply src/Momento.Sdk/Responses/CacheDeleteResponse.cs:5]: Thanks for the context!

**Review by malandis — APPROVED** [review]

**Pull request merged**